test/fixtures/poro.rb

```diff
@@ -1,6 +1,10 @@
 class Model
   FILE_DIGEST = Digest::MD5.hexdigest(File.open(__FILE__).read)
 
+  def self.create(params)
+    @attributes = params
+  end
```

Comment:
It's because I started to write some [tests for deserialization](https://github.com/rails-api/active_model_serializers/pull/950/files#diff-81a0ce879bbbddb9593f3e0faa8c16bbR9)